index.html

```diff
@@ -105,27 +105,101 @@
   </section>
 
   <section>
-    <h2>Network Error Logging</h2>
-
+    <h2>Conformance requirements</h2>
+    <p>All diagrams, examples, and notes in this specification are non-normative, as are all sections explicitly marked non-normative.  Everything else in this specification is normative.</p>
+    <p>The key words "MUST", "MUST NOT", "REQUIRED", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in the normative parts of this document are to be interpreted as described in [[!RFC2119]].</p>
+    <p>Requirements phrased in the imperative as part of algorithms (such as "strip any leading space characters" or "return false and abort these steps") are to be interpreted with the meaning of the key word ("must", "should", "may", etc) used in introducing the algorithm.</p>
+    <p>Some conformance requirements are phrased as requirements on attributes, methods or objects. Such requirements are to be interpreted as requirements on the user agent.</p>
+    <p>Conformance requirements phrased as algorithms or specific steps may be implemented in any manner, so long as the end result is equivalent. (In particular, the algorithms defined in this specification are intended to be easy to follow, and not intended to be performant.)</p>

```

Comment:
I coped this mostly verbatim from the BEACON spec that you mentioned in #68; let me know if the text needs to be more custom to this spec

Reply:
lgtm